scripts/cnv_wdl/somatic/cnv_somatic_funcotate_seg_workflow.wdl

```diff
@@ -0,0 +1,151 @@
+workflow CNVFuncotateSegmentsWorkflow {
```

Comment:
This should have some comments at the top to help with input arguments / valid types ala the Funcotator WDL:

```
# Run Funcotator on a set of called variants from Mutect 2.
#
# Description of inputs:
#
#   Required:
#     String gatk_docker                  - GATK Docker image in which to run
#     File ref_fasta                      - Reference FASTA file.
#     File ref_fasta_index                - Reference FASTA file index.
#     File ref_fasta_dict                 - Reference FASTA file sequence dictionary.
#     File variant_vcf_to_funcotate       - Variant Context File (VCF) containing the variants to annotate.
#     File variant_vcf_to_funcotate_index - Index file corresponding to the input Variant Context File (VCF) containing the variants to annotate.
#     String reference_version            - Version of the reference being used.  Either `hg19` or `hg38`.
#     String output_file_name             - Path to desired output file.
#     String output_format                - Output file format (either VCF or MAF).
#     Boolean compress				      - Whether to compress the resulting output file.
#     Boolean use_gnomad                  - If true, will enable the gnomAD data sources in the data source tar.gz, if they exist.
#
#   Optional:
#     interval_list                       - Intervals to be used for traversal.  If specified will only traverse the given intervals.
#     data_sources_tar_gz                 - Path to tar.gz containing the data sources for Funcotator to create annotations.
#     transcript_selection_mode           - Method of detailed transcript selection.  This will select the transcript for detailed annotation (either `CANONICAL` or `BEST_EFFECT`).
#     transcript_selection_list           - Set of transcript IDs to use for annotation to override selected transcript.
#     annotation_defaults                 - Annotations to include in all annotated variants if the annotation is not specified in the data sources (in the format <ANNOTATION>:<VALUE>).  This will add the specified annotation to every annotated variant if it is not already present.
#     annotation_overrides                - Override values for annotations (in the format <ANNOTATION>:<VALUE>).  Replaces existing annotations of the given name with given values.
#     gatk4_jar_override                  - Override Jar file containing GATK 4.0.  Use this when overriding the docker JAR or when using a backend without docker.
#     funcotator_extra_args               - Extra command-line arguments to pass through to Funcotator.  (e.g. " --exclude-field foo_field --exclude-field bar_field ")
#
# This WDL needs to decide whether to use the ``gatk_jar`` or ``gatk_jar_override`` for the jar location.  As of cromwell-0.24,
# this logic *must* go into each task.  Therefore, there is a lot of duplicated code.  This allows users to specify a jar file
# independent of what is in the docker file.  See the README.md for more info.
```

Reply:
Done